doc/config.toml

```diff
@@ -0,0 +1,22 @@
+baseURL = "http://localhost"
+languageCode = "en-us"
+title = "The Things Network Stack for LoRaWAN - V3"
+theme = "hugo-theme-techdoc"
+
+enableGitInfo = true
+
+[frontmatter]
+date = [":git", "date", "publishDate", "lastmod"]
+lastmod = [":git", "lastmod", "date", "publishDate"]
+publishDate = ["publishDate", "date"]
+expiryDate = ["expiryDate"]
+
+[Params]
+    # Souce Code repository section
+    description = "Open-source LoRaWAN network stack suitable for large, small, global and geo-distributed deployments."
```

Comment:
>The Things Network Stack for LoRaWAN is an open-source LoRaWAN network stack suitable for large, global and geo-distributed public and private networks as well as smaller networks.

Reply:
It appears just below the title.
This is a theme thing though so we can change it later.